<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>curserWidget</class>
 <widget class="QWidget" name="curserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="curser_verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="curserPos_label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Curser Position (x,y,z)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="cursorPos_frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="curserX_label">
        <property name="text">
         <string>#</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="curserY_label">
        <property name="text">
         <string>#</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="curserZ_label">
        <property name="text">
         <string>#</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="curserIntensity_label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Intensity (min,max,cur):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="curserIntensity_frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="minIntensity_label">
        <property name="text">
         <string>#</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxIntensity_label">
        <property name="text">
         <string>#</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="curIntensity_label">
        <property name="text">
         <string>#</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Window Value</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="winVal_label">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="winVal_slider">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Level Value</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="levVal_label">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="levVal_slider">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
